v22.1/query-behavior-troubleshooting.md

```diff
@@ -17,7 +17,7 @@ For a developer-centric overview of optimizing SQL statement performance, see [O
 
 Use the [slow query log](logging-use-cases.html#sql_perf) or DB Console to detect slow queries in your cluster.
 
-High latency SQL statements are displayed on the [Statements](ui-statements-page.html) page of the DB Console.
+High latency SQL statements are displayed on the [Statements](ui-statements-page.html) page of the DB Console. To help you focus on such statements, when you create a [diagnostics bundle](ui-statements-page.html#diagnostics) you can specify that the diagnostics are collected only when a statement fingerprint exceeds a certain latency.
```

Comment:
How about this?

High latency SQL statements can be identified on the Statements page of the DB console. You can collect richer diagnostics of a high latency statement by creating a diagnostics bundle when a statement fingerprint exceeds a certain latency.